xtas/tasks/single.py

```diff
@@ -282,7 +282,7 @@ def sentiwords_tag(doc, output="bag"):
         return counts
 
     elif output == "tokens":
-        return [ngram if polarity == 0 else (ngram, polarity)
```

Comment:
I guess I don't see how that has anything to do with this. The result would get attached as a child document, right? There'd be some None's in there so it would be bigger, true, but it would still index the entire child document?